android-components/components/service/contile/src/main/java/mozilla/components/service/contile/ContileTopSitesProvider.kt

```diff
@@ -169,9 +173,16 @@ class ContileTopSitesProvider(
         }
     }
 
+    private fun getCachedTopSitesIfValid(shouldUseServerMaxAge: Boolean = false) =
```

Comment:
nit: I actually think it might make usages of this function more clear to remove the default parameter and make every callsite explicit